src/styles/dayCard.css

```diff
@@ -8,10 +8,13 @@
   flex-direction: column;
   justify-content: center;
   align-items: center;
-  gap: 1rem;
+  gap: 0.8rem;
```

Comment:
read how to create variables in css.

Reply:
now not used as styling is shifted to reactstrap